services/addSort.js

```diff
@@ -0,0 +1,14 @@
+import { sortByNumbers, sortName } from './sorting';
+
+export default function addSort(array, filter) {
```

Comment:
```suggestion
export default function addSort(array, sortOptions) {
```
rename the variable to better reflect what it contains